TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/cli.py

```diff
@@ -93,42 +98,75 @@ def start_replayer_cmd(ctx):
 def backfill_group(ctx):
     """All actions related to historical/backfill data migrations"""
     if ctx.env.backfill is None:
-        raise ValueError("Backfill migration is not set")
+        raise click.UsageError("Backfill migration is not set")
+
 
+@backfill_group.command(name="describe")
+@click.pass_obj
+def describe_backfill_cmd(ctx):
+    click.echo(logic_backfill.describe(ctx.env.backfill, as_json=ctx.json))
 
-@backfill_group.command(name="create-migration")
+
+@backfill_group.command(name="create")
```

Comment:
I'm not sure I understand as a user what a 'create' does in this context - is this the right name vs 'prepare' or 'configure'? How is this different from 'start'?

Reply:
Yes, I think it could be equivalent to 'prepare'.  I'm not sure which is better.

It's different than `start` because the end state of this command should _not_ be "data is flowing", but just "ready to start (i.e. make data flow)". Specifically, an OSI pipeline needs to be created. This will be a no-op for RFS at this point (though in the future, we could use this to take the config file params and create a task definition, for instance).

Reply:
I suppose getting this in front of users will generate more feedback and we can iterate on this if need be.